0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -148,46 +171,75 @@ Content-Type: application/json
 
 ### Payment Setup
 
-The sender uses the receiver details to create the ILP packet:
+The sender uses the receiver details to create the PSK2 payment:
 
-* The `account` in the ILP packet is the `destination_account` provided by the receiver
-* The `amount` is determined by the sender:
+* The `destination_account` should be used as the PSK2 destinationAccount.
+* The `shared_secret` should be used as the PSK2 sharedSecret.
+* If present, the `balance.maximum` SHOULD be used as the PSK2 chunked payment amount.
 
-    * The sender uses the `ledger_info.currency_scale` to determine the integer amount to include in the ILP packet (e.g. if they want the receiver to receive 10 units of `ledger_info.currency_code` on their ledger, a `ledger_info.currency_scale` value of 2 would indicate they should set the `amount` to 1000). In cases where the sender knows the integer amount the receiver should get, for example if that amount was requested by the receiver out of band, the sender does not need to use `ledger_info.currency_code` and `ledger_info.currency_scale`
-    * The sender SHOULD NOT set an amount greater than the `maximum_destination_amount` or lower than the `minimum_destination_amount`, as this will be rejected by the receiver
+In a UI, the `asset_info` and `receiver_info` objects (if present) can be used for display purposes. These objects can be manipulated by the receiver in any way, so amounts SHOULD be displayed in source units when possible.
 
-* The `data` is encoded using the [Pre-Shared Key protocol](../0016-pre-shared-key/0016-pre-shared-key.md):
+Note that the sender can send as many PSK2 payments as they want using the same receiver info. The sender SHOULD query the receiver again once the time indicated in the [`Cache-Control` header](#response-headers) has passed.
 
-    * The key is the `shared_secret` provided by the receiver
-    * Additional data may be included in the PSK details. SPSP data MUST be JSON.
-    * Private PSK headers SHOULD include `Content-Type: application/json` and `Content-Length: <byte length of data>`
+## Appendix A: Payment Pointer
 
-Note that the sender can send as many payments as they want using the same receiver info. The sender SHOULD query the receiver again once the time indicated in the [`Cache-Control` header](#response-headers) has passed.
+This is the recommended way to identify an SPSP receiver, and is intended to be the main form of identifier that users on Interledger will interact with. It can be used as a persistent identifier for a person or as a temporary identifier to represent an invoice, much like a bitcoin address.
 
-## Appendix A: (Optional) Webfinger Discovery
+The payment pointer is in the form `$example.com/bob` (A payment pointer with no path is also acceptable, i.e. `$bob.example.com`). This is converted into an SPSP URI, by removing the `$` and replacing it with `https://`. If the payment pointer has no path, then a path of `/.well-known/pay` is added.
```

Comment:
It seems a bit strange that the resolution behavior depends on whether there's a path. Is there a reason not to do something like webfinger where you query the endpoint with the whole identifier?

Reply:
yeah, that makes it very difficult to have one identifier that redirects to another. For example if my SPSP receiver is hosted on `$sharafian.example.com` and I set `$sharafian.com` to redirect there, then example.com will end up with the request `https://sharafian.example.com/.well-known/pay?resource=$sharafian.com`, which it won't know what to do with. This was a constant annoyance when we were using webfinger and ILP kit, so I'd be happy if we don't have to re-introduce it.

Having a default path isn't very hard behavior to implement, and I think it isn't so strange to say you use a hard-coded default unless there's a specific path specified in the identifier.